blackbox-exporter.json

```diff
@@ -64,7 +64,7 @@
   "panels": [
     {
       "columns": [],
-      "datasource": "${DS_PROMETHEUS}",
+      "datasource": "${DS_LOCALHOST}",
```

Comment:
same here, the datasource on my side is:

```
      "datasource": "${DS_PROMETHEUS}",
```

i don't exactly remember how i do the export here... the [dashboard update procedure](https://github.com/hiromipaw/grafana-dashboards/blob/master/README.md#updating-dashboards) is a bit unreliable to be honest.

but the original version seemed to have worked for me. that's on nbg-1 though, maybe the datasources names are different on the new server? in that case i'd make sure they match the expected datasource name here instead of rewriting all dashboards... :)

the datasource, of course, should also be in puppet - that's one of the parts that's not fully puppetized IIRC.